doc/source/gallery/inset_map.ipynb

```diff
@@ -0,0 +1,442 @@
+    "ax = nybb.plot()\n",
+    "\n",
+    "# Now adding an inset axis\n",
+    "iax = ax.inset_axes([0.05, 0.70, 0.25, 0.25], xticks=[], yticks=[])\n",
+    "# Additional data may now be plotted on this axis\n",
+    "# ex. new_york_state.plot(ax=iax)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using [`mpl_toolkits.axes_grid1.inset_locator.inset_axes`](https://matplotlib.org/stable/api/_as_gen/mpl_toolkits.axes_grid1.inset_locator.inset_axes.html)\n",
+    "\n",
+    "This separate `inset_axes()` function provided by `matplotlib` is intended to be used in a similar way to the `inset_axes()` method of the Axes class. However, it may also (optionally) be paired with [`mark_inset()`](https://matplotlib.org/stable/api/_as_gen/mpl_toolkits.axes_grid1.inset_locator.mark_inset.html) to connect the inset axes to the parent axes, in the case where the inset axes are showing additional detail of a small area of the parent axes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geopandas as gpd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import mpl_toolkits.axes_grid1.inset_locator as mil\n",
+    "import geodatasets as gda"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Creating a Parent and Inset Axis\n",
+    "At a high level, creating a simple parent and inset axis occurs with more-or-less the same workflow as above.\n",
+    "\n",
+    "However, this `inset_axes()` function is very different from the `inset_axes()` method of the Axes class; most notably: \n",
+    "\n",
+    "- It is applied as a standalone function (via `mpl_toolkits.axes_grid1.inset_locator.inset_axes()`) rather than as a method of an Axes object (like `ax.inset_axes()`). As such, the parent axes must be passed as the first argument.\n",
+    "\n",
+    "- Instead of providing a 4-tuple of values specifying the bottom-left corner and width and height of the inset axes, it requires separate values for `width` and `height`, as well as a `loc` keyword similar to how `Legend` objects are placed. \n",
+    "\n",
+    "  - If integers or floats are passed to width and height, they are interpreted as *inches*. If strings are passed, they are interpreted as percentages of the parent axes, or to the bounding box specified under `bbox_to_anchor`.\n",
+    "\n",
+    "  - The `loc` keyword is similar to the `loc` keyword of `Legend` objects. It can be a string or an integer. See [this documentation](https://matplotlib.org/stable/api/legend_api.html#matplotlib.legend.Legend) for more details.\n",
+    "\n",
+    "  - Advanced positioning is also possible via `bbox_to_anchor`, and `bbox_transform`, just like `Legend`s. See [this documentation](https://matplotlib.org/stable/api/_as_gen/mpl_toolkits.axes_grid1.inset_locator.inset_axes.html) for more details.\n",
```

Comment:
```suggestion
    "  - The `loc` keyword is similar to the `loc` keyword of `Legend` objects. It can be a string or an integer. See [the documentation](https://matplotlib.org/stable/api/legend_api.html#matplotlib.legend.Legend) for more details.\n",
    "\n",
    "  - Advanced positioning is also possible via `bbox_to_anchor`, and `bbox_transform`, just like `Legend`s. See [the documentation](https://matplotlib.org/stable/api/_as_gen/mpl_toolkits.axes_grid1.inset_locator.inset_axes.html) for more details.\n",
```